x/pairing/keeper/unstaking.go

```diff
@@ -52,13 +52,22 @@ func (k Keeper) UnstakeEntry(ctx sdk.Context, provider bool, chainID string, cre
 	if existingEntry.Deadline < holdBlocks {
 		existingEntry.Deadline = holdBlocks
 	}
+	details := map[string]string{"stake_entry": existingEntry.String()}
+	utils.LogLavaEvent(ctx, logger, types.UnstakeCommitNewEventName(provider), details, " Appending new unstake entry (removing stake entry).")
 	return k.epochStorageKeeper.AppendUnstakeEntry(ctx, stake_type, existingEntry)
 }
 
 func (k Keeper) CheckUnstakingForCommit(ctx sdk.Context) error {
 	// this pops all the entries that had their deadline pass
 	unstakingEntriesToCredit := k.epochStorageKeeper.PopUnstakeEntries(ctx, epochstoragetypes.ProviderKey, uint64(ctx.BlockHeight()))
+
 	if unstakingEntriesToCredit != nil {
+		details := map[string]string{"unstake_entities_type": epochstoragetypes.ProviderKey}
+		for i, unstakeEntry := range unstakingEntriesToCredit {
+			details["unstake_entry_"+strconv.Itoa(i)] = unstakeEntry.String()
+		}
+		utils.LogLavaEvent(ctx, k.Logger(ctx), types.UnstakeCommitNewEventName(true), details, " Crediting "+epochstoragetypes.ProviderKey+" unstake entries.")
```

Comment:
there is already an event inside k.creditUnstakingEntries

Reply:
revert what you added

Reply:
done